### PR TITLE
Add user info

### PR DESCRIPTION
**Pull request by apratt101300**
Added additional information about each user to their profile page. 

To test:
1. Make migrations/migrate
2. In the admin console, create a new badge
3. In the admin console, create a new Profile, which will have a one-to-one relation to a user. Make sure the user it's based on 
   has a first name, last name, and email
4. In the application, click on the link to any user's username to view their information

@John151 @Nelwell @abdiawali @PickleEric

### CODE REVIEW

File: lmn/models.py [hunk 1]
```diff
@@ -58,3 +59,45 @@ class Note(models.Model):
...
+    def save(self, *args, **kwargs):
+        old_profile = Profile.objects.filter(pk=self.pk).first()
+        if old_profile and old_profile.profile_image:
+            if old_profile.profile_image != self.profile_image:
+                self.delete_image(old_profile.profile_image)
+
+        super().save(*args, **kwargs)
+
+
+    def delete_image(self, image):
```
Comment (PickleEric) [inline lmn/models.py:88]: Maybe Add a confirm message before deletion?

**Review by PickleEric — APPROVED** [review]
code looks great

File: lmn/models.py (hunk 1 above)
Comment (apratt101300) [reply lmn/models.py:88]: I will add a confirm delete message later on. These methods ensure that the images themselves are deleted from the media directory when a profile image is changed or deleted. The user doesn't have the ability to edit their profile yet, but when that feature is implemented, we'll definitely want a message to confirm delete.  Thanks for the reminder!

**Comment by Nelwell** [comment]
Nice! feature works and code is clean. May add a way to make a user profile to auto-create when making a new account later.

**Pull request merged**